oam/install_scripts/syslogSetup.sh

```diff
@@ -180,23 +191,23 @@ if [ ! -z "$syslog_conf" ] ; then
 		      group=`stat -c "%U %G" /var/log/syslog | awk '{print $2}'`
 		fi
 		
-		//set permissions
-		$SUDO chown $user:$group -R /var/log/mariadb > /dev/null 2>&1
+		# set permissions
+		chown $user:$group -R /var/log/mariadb > /dev/null 2>&1
 		
 		if [ $rsyslog7 == 1 ]; then
-			sed -i -e s/groupname/$group/g ${columnstoreSyslogFile7}
-			sed -i -e s/username/$user/g ${columnstoreSyslogFile7}
-			
-			$SUDO rm -f /etc/rsyslog.d/49-columnstore.conf
-			$SUDO cp  ${columnstoreSyslogFile7} ${syslog_conf}
+			 rm -f /etc/rsyslog.d/49-columnstore.conf
+			 cp  ${columnstoreSyslogFile7} ${syslog_conf}
+
+			 sed -i -e s/groupname/$groupname/g ${syslog_conf}
```

Comment:
Just to save some time on this review, I'll just say here that wherever the change was from 'sudo something' to 'something', and 'something' does need root perms, it won't work now when running as not-root.